src/Compatibility/Microsoft.DotNet.ApiSymbolExtensions/Filtering/CompositeSymbolFilter.cs

```diff
@@ -9,19 +9,33 @@ namespace Microsoft.DotNet.ApiSymbolExtensions.Filtering
     /// Implements the composite pattern, group the list of <see cref="ISymbol"/> and interact with them
     /// the same way as a single instance of a <see cref="ISymbol"/> object.
     /// </summary>
-    public sealed class CompositeSymbolFilter : ISymbolFilter
+    public sealed class CompositeSymbolFilter(CompositeSymbolFilterMode mode = CompositeSymbolFilterMode.And,
```

Comment:
I saw that you updated `default` to `CompositeSymbolFilterMode.And` in the tests. What if we make this a required parameter instead, without providing a default?

Reply:
Yeah, I'm good with that, will make that change.